src/VirtualClient/VirtualClient.Actions/DiskPerformanceWorkloadExecutor.cs

```diff
@@ -397,17 +397,20 @@ protected virtual IEnumerable<DiskPerformanceWorkloadProcess> CreateWorkloadProc
             return this.Logger.LogMessage($"{this.GetType().Name}.CreateProcesses", telemetryContext, () =>
             {
                 List<DiskPerformanceWorkloadProcess> processes = new List<DiskPerformanceWorkloadProcess>();
-                string testedInstance = this.DiskFilter;
 
                 if (string.Equals(processModel, WorkloadProcessModel.SingleProcess, StringComparison.OrdinalIgnoreCase))
                 {
-                    processes.Add(this.CreateWorkloadProcess(executable, commandArguments, testedInstance, disks.Select(disk => disk).ToArray()));
+                    // Example Metric Categorization
+                    // SingleProcess,BiggestSize,16
+                    processes.Add(this.CreateWorkloadProcess(executable, commandArguments, $"{WorkloadProcessModel.SingleProcess},{this.DiskFilter},{disks.Count()}", disks.ToArray()));
                 }
                 else if (string.Equals(processModel, WorkloadProcessModel.SingleProcessPerDisk, StringComparison.OrdinalIgnoreCase))
                 {
+                    // Example Metric Categorization
+                    // SingleProcessPerDisk,BiggestSize,16
                     processes.AddRange(new List<DiskPerformanceWorkloadProcess>(disks.Select(disk =>
                     {
-                        return this.CreateWorkloadProcess(executable, commandArguments, testedInstance, disk);
+                        return this.CreateWorkloadProcess(executable, commandArguments, $"{WorkloadProcessModel.SingleProcessPerDisk},{this.DiskFilter},{disks.Count()}", disk);
```

Comment:
Is this intentional that you use single disk but add diskcount?

Reply:
It was but I guess it will not skew anything if we have it as SingleProcessPerDisk,DiskFilter,1

Reply:
Changed it to use "1" instead of disk count.

Reply:
Actually I think your way is better, it tells you the total amount